src/explicitode.jl

```diff
@@ -63,22 +62,22 @@ the current system state (the initial conditions), and `jetcoeffs!`
 computes recursively the high-order derivates back into `x`.
 
 """
-function jetcoeffs!{T<:Real, U<:Number}(eqsdiff!, t0::T, x::Vector{Taylor1{U}}, 
-        dx::Vector{Taylor1{U}}, xaux::Vector{Taylor1{U}}, vT::Vector{T})
+function jetcoeffs!{T<:Real, U<:Number}(eqsdiff!, t::Taylor1{T}, x::Vector{Taylor1{U}}, 
+        dx::Vector{Taylor1{U}}, xaux::Vector{Taylor1{U}})
```

Comment:
Note: after merging #31, there won't be need for `xaux`; the idea is to have for the scalar case:
```julia
jetcoeffs!{T<:Real, U<:Number}(eqsdiff, t::Taylor1{T}, x::Vector{Taylor1{U}})
```
and for the vectorial case:
```julia
jetcoeffs!{T<:Real, U<:Number}(eqsdiff!, t::Taylor1{T}, x::Vector{Taylor1{U}}, dx::Vector{Taylor1{U}})
```

Reply:
I meant
```julia
jetcoeffs!{T<:Real, U<:Number}(eqsdiff, t::Taylor1{T}, x::Taylor1{U})
```
for the scalar case